waku/v2/protocol/waku_message/message.nim

```diff
@@ -12,7 +12,8 @@ else:
 import
   ../../utils/time
 
-const MaxWakuMessageSize* = 1024 * 1024 # In bytes. Corresponds to PubSub default
+const
+  MaxWakuMessageSize* = 1024 * 1024 # 1 Mibytes. Corresponds to PubSub default
```

Comment:
```suggestion
const MaxWakuMessageSize* = 1024 * 1024 # 1 Mibytes. Corresponds to PubSub default
```

Reply:
As I plan to restore the max topic length constants, I opted not to restore the single line const :)